wallaby.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (wallaby) {
```

Comment:
Curious if you intended to commit this file? Wallaby.js is not free software, so we'd prefer not to use it here.

Reply:
No, my mistake I didn't mean to commit that. I was testing out wallaby while doing this PR.